File: chatwoot/python/src/AddNewCannedResponseToAccountExample.py
import json
from datetime import date, datetime
from pprint import pprint

from chatwoot_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"userApiKey": "USER_API_KEY"},
)

with ApiClient(configuration) as api_client:
    canned_response_create_update_payload = models.CannedResponseCreateUpdatePayload(
        content=None,
        short_code=None,
    )

    try:
        response = api.CannedResponsesApi(api_client).add_new_canned_response_to_account(
            account_id=None,
            canned_response_create_update_payload=canned_response_create_update_payload,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling CannedResponsesApi#add_new_canned_response_to_account: %s\n" % e)

File: chatwoot/ruby/src/AddNewCannedResponseToAccountExample.rb
require "json"
require "chatwoot_client"

ChatwootClient.configure do |config|
    config.api_key["api_access_token"] = "USER_API_KEY"
end

canned_response_create_update_payload = ChatwootClient::CannedResponseCreateUpdatePayload.new

begin
    response = ChatwootClient::CannedResponsesApi.new.add_new_canned_response_to_account(
        0, # account_id
        canned_response_create_update_payload, # data
    )

    p response
rescue ChatwootClient::ApiError => e
    puts "Exception when calling CannedResponsesApi#add_new_canned_response_to_account: #{e}"
end
